flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskMailboxTestHarness.java

```diff
@@ -136,6 +136,17 @@ public boolean processSingleStep() throws Exception {
         return false;
     }
 
+    public boolean processSingleMailboxLoop() throws Exception {
+        if (streamTask.mailboxProcessor.isMailboxLoopRunning()) {
+            return streamTask.runSingleMailboxLoop();
+        }
+        return false;
+    }
+
+    public MailboxExecutor getMainMailboxExecutor() {
+        return streamTask.mailboxProcessor.getMainMailboxExecutor();
+    }
+
```

Comment:
Seems unused? also the `StreamTask#runSingleMailboxLoop` and `MailboxProcessor#runSingleMailboxLoop`

Reply:
left overs from previous version. removed